src/Compilers/VisualBasic/Portable/Binding/BinderFactory.vb

```diff
@@ -387,8 +387,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                                  SyntaxKind.GetAccessorStatement,
                                  SyntaxKind.AddHandlerAccessorStatement,
                                  SyntaxKind.RemoveHandlerAccessorStatement,
-                                 SyntaxKind.RaiseEventAccessorStatement
-                                ' Default values are not valid (and not bound) for lambda parameters or property accessors
+                                 SyntaxKind.RaiseEventAccessorStatement,
+                                 SyntaxKind.OperatorStatement
```

Comment:
@AlekseyTs checked the call stack. Basically `ParseExpressionCore(bailIfFirstTokenRejected)` is false when parsing optional parameters, so it returns an empty expression instead of `Nothing`. In all checks through the code base, we check whether the default value of the parameter is `Nothing` or not. I'm changing it now, but is there any value of using an empty expression syntax here?